cmd/secrets-provider/main.go

```diff
@@ -10,7 +10,8 @@ import (
 	"github.com/cyberark/conjur-authn-k8s-client/pkg/authenticator"
 	authnConfigProvider "github.com/cyberark/conjur-authn-k8s-client/pkg/authenticator/config"
 
-	"github.com/cyberark/secrets-provider-for-k8s/pkg/log"
+	"github.com/cyberark/conjur-authn-k8s-client/pkg/log"
```

Comment:
please move this upwards with the rest of the authn-client imports

Reply:
done.